### ISSUE TITLE
Allows other apps to share one or more photos to Aves

### ISSUE THREAD

**Issue by petazonk** [deckerst/aves#1151]
**Is your feature request related to a problem? Please describe.**
Sometimes I find photos in other apps and I want to tag them in Aves but at the moment, I have to manually find the photos in Aves and then tag them which is not always easy.

**Describe the solution you'd like**
Let's say I find a bunch of photos from another gallery (an example might be incorrect dates and times) but that gallery can't tag or edit metadata. I would like to share these photos with Aves such that I can tag them quickly in Aves. Another example could be correcting GPS locations.

**Describe alternatives you've considered**
I have not been able to find any other way to quickly share photos to Aves.

**Comment by deckerst** [comment]
Do you expect to see them in the Collection page, or scrollable one by one in the full screen viewer?

**Comment by petazonk** [comment]
They would need to be a collection such that you could select them all or
multi select them. Then you could perform a batch operation on them. From
the collection you could go to full screen and scroll through them. Swiping
back would take you back to the collection.

**Comment by deckerst** [comment]
One thing to bear in mind: some apps share images via a custom content provider and in that case the receiving app cannot edit the items, because it does not have the standard uri/path to access them.

If you share a single image to Aves, and see in the item `Info` page that the URI is something like `content://media/external/...` it's all good. Otherwise it means the sending app uses a custom content provider.

**Comment by petazonk** [comment]
Well you can only work with what you have. My main use is sharing photos from Google Gallery, which does local face tagging, so I could utilise the face tag data by sharing with Aves. Also both Google Gallery and Photos allow you to mark photos as a favourite but unfortunately it seems to be a propriety tag/favourite and thus not accessible from other apps.

**Comment by k3b** [comment]
@deckerst 
> Do you expect to see them in the Collection page, or scrollable one by one in the full screen viewer?

It may be easier to implement this feature if the from-other-app-selected-images that aves receives are treated the same way as they where multiselected inside aves. 

Example usecase: Add geo-coordinates to holiday photos:

* Find fotos without geo-info in the other app
* Select those photos that have roughly the same location (i.e. photos of my holidy hotel)
* "send" them to aves
* in aves batch-setGeo (either from a hotel-photo that has geo data or else via picking a location from the map).

If i also want to tag the selected with the keyword "hotel" i can send them again form the external app.

**Comment by petazonk** [comment]
Yeah that makes sense in that once you get them into Aves you could add a
tag to get them from Aves again easily for whatever is needed.

**Comment by deckerst** [comment]
Another interesting use case: transfer items to the Secure Folder on Samsung devices. After sharing multiple items to Aves installed in the secure space, items could be copied to the secure storage.
